.env.sample

```
@@ -0,0 +1 @@
+# AMPLITUDE_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)